### PR TITLE
Use verb "to set up" instead of noun "setup"

### PR DESCRIPTION
**Pull request by osg-grafana**

### CODE REVIEW

**Review by eleijonmarck — APPROVED** [review]

File: docs/sources/enterprise/saml/set-up-saml-with-okta.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 ---
-title: 'Setup SAML authentication with Okta in Grafana'
+title: 'Set up SAML authentication with Okta in Grafana'
 menuTitle: 'SAML authentication with Okta'
-description: 'This is a guide to setup SAML authentication with Okta in Grafana'
+description: 'This is a guide to set up SAML authentication with Okta in Grafana'
 keywords: ['grafana', 'saml', 'documentation', 'saml-auth', 'enterprise']
```
Comment (osg-grafana) [inline docs/sources/enterprise/saml/set-up-saml-with-okta.md:5]:
```suggestion
keywords: ['grafana', 'saml', 'documentation', 'saml-auth', 'enterprise']
aliases = ["/docs/enterprise/latest/saml/setup-saml-with-okta/"]
```

Comment (osg-grafana) [reply docs/sources/enterprise/saml/set-up-saml-with-okta.md:5]: @achatterjee-grafana, is this alias correct?

**Comment by osg-grafana** [comment]
I need to still check if there are any references from other docs sets to this section, for example in Cloud.

File: docs/sources/enterprise/saml/set-up-saml-with-okta.md (hunk 1 above)
Comment (osg-grafana) [reply docs/sources/enterprise/saml/set-up-saml-with-okta.md:5]: Actually, this is in `next` and not `latest`, so no alias needed.

**Pull request merged**